vpr/src/draw/intra_logic_block.cpp

```diff
@@ -237,11 +237,30 @@ static void draw_internal_load_coords(int type_descrip_index, t_pb_graph_node* p
             /* Find the number of instances for each child pb_type. */
             int num_pb = mode.pb_type_children[j].num_pb;
 
+            int num_blocks = num_pb * num_children;
+
+            // determine an optimal number of columns
```

Comment:
Explain this algorithm.  Here's what I think it is:
Find a number or rows and columns that will be as nearly equal as possible (to make a squarish 2D grid) and where num_rows * num_columns gives us exactly the right number of blocks.

Reply:
Avoid using the term optimal, as that means the provably best -- can't prove this is the best visually :).